### DISCUSSION TITLE
error please upgrade your co pilot extension

### DISCUSSION THREAD

**Discussion by PhilTs454** [community/community#18766] [Copilot Conversations] [answered]
i used the co pilot for a couple of days and suddenly this error happened "error please upgrade your co pilot extension to continue using this service".
tried updating the co pilot and the vlc but nothing works
anyone have a solution?

**Comment by davecheney** [comment, marked as answer]
@PhilTs454 please upgrade to version 1.29 (or later) of the VSCode extension, or version 1.1.25 (or later) of the Intellij extension.

**Reply by PhilTs454** [reply]
thank you

**Comment by PhilTs454** [comment]
Thank you very much for your answer
